osclib/repair_command.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from osc import oscerr
+from osc.core import change_review_state
+from osc.core import get_request
+from osclib.request_finder import RequestFinder
+
+def _is_int(x):
+    return isinstance(x, int) or x.isdigit()
+
+class RepairCommand(object):
+
+    def __init__(self, api):
+        self.api = api
+
+    def repair(self, request):
+        reviews = []
+        reqid = str(request)
+        req = get_request(self.api.apiurl, reqid)
+
+        if not req: 
+            raise oscerr.WrongArgs('Request {} not found'.format(reqid))
+
+        if req.state.name != 'review':
+            print('Request "{}" is not in review state'.format(reqid))
+            return
+
+        for review in req.reviews:
+            if re.match(r'.*:Staging:(.*)$', str(review.by_project)) and review.state == 'new':
+                reviews.append(review.by_project)
```

Comment:
only as a curiosity, the last three lines can be written as:

```Python
review = [r.by_project for r in req.review if ':Staging:' in str(r.by_project) and r.state == 'new']
```

Not sure that we need this `str()`